frontend/server/src/Controllers/Session.php

```diff
@@ -197,6 +196,7 @@ private static function getCurrentSessionImpl(\OmegaUp\Request $r): array {
             'is_admin' => \OmegaUp\Authorization::isSystemAdmin(
                 $currentIdentity
             ),
+            'all_identities' => $allIdentities,
```

Comment:
```suggestion
            'associated_identities' => $associatedIdentities,
```

es mejor usar el adjetivo "associated" porque "all" puede interpretarse como "todas las que existen" o "todas las del usuario".

Reply:
Listo